tests/pyccel/scripts/return_numpy_arrays.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+def single_return(x: 'int'):
+    return np.ones(x)
```

Comment:
See issue #1130 . I think you will need to name the arrays before calling return for the tests to work